roles/tools/tasks/write_pmm_kubectl_config.yaml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Checking for .kube config file
```

Comment:
@aaronplagge why is this here when it is already elsewhere https://github.com/cezmunsta-org/gascan-ansible/pull/11/files#diff-ae6a42d78cbb5225d5d0a1c72f7a78fec4178fa0a1591becbf424588651f4ff1R17-R22 ?

Reply:
This is here as if the .kube/config file was written out previously it will change then use a name of .kube/config_<PMM_ENV_NAME> to write it out to file therefore not overwriting the existing .kube/config file.